Packs/SecneurXThreatFeeds/Integrations/SecneurXThreatFeeds/SecneurXThreatFeeds.py

```diff
@@ -47,68 +102,121 @@ def parseIndicators(feedJson: Dict) -> Any:
-                            "verdict": "Malicious",
-                            "tags": tagList,
-                            "modified": modifiedValue,
-                            "reportedby": reportByValue
-                        }
-                        indicatorList.append(indicatorObj)
-
-                    except Exception as e:
-                        demisto.error(e)
+                        if SNXIocParse.SNX_MALWARE_TYPE_KEY in ioc_data.keys():
+                            tagList = ioc_data[SNXIocParse.SNX_MALWARE_TYPE_KEY]
+                        if SNXIocParse.SNX_DESCRIPTIONS_KEY in ioc_data.keys():
+                            descriptionValue = ioc_data[SNXIocParse.SNX_DESCRIPTIONS_KEY]
+                    else:
+                        continue
+
+                    indicatorObj = {}
+                    if indicatorValue is not None:
+                        indicatorObj["value"] = indicatorValue
+                    else:
+                        continue
+                    if indicatorType is not None:
+                        indicatorObj["type"] = indicatorType
+                    else:
+                        continue
+                    indicatorObj["rawJson"] = ioc_data
+                    indicatorObj["fields"] = {
+                        "indicatoridentification": indicatorIdValue,
+                        "verdict": SNXIocParse.SNX_MALICIOUS_KEY,
+                        "tags": tagList,
+                        "reportedby": reportByValue
+                    }
+                    if createdValue is not None:
+                        indicatorObj["fields"]["firstseenbysource"] = createdValue
+                    if modifiedValue is not None:
+                        indicatorObj["fields"]["modified"] = modifiedValue
+                    if confidenceValue is not None:
+                        indicatorObj["fields"]["confidence"] = confidenceValue
+                    if descriptionValue is not None:
+                        indicatorObj["fields"]["description"] = descriptionValue
+                    indicatorList.append(indicatorObj)
+
+                except Exception as e:
+                    raise DemistoException(e)
 
     except Exception as e:
-        demisto.error(e)
+        raise DemistoException(e)
     return indicatorList
 
 
+def parse_ioc_values(ioc_data):
+    patternKey = None
+    patternValue = None
+    try:
+        patternInfo = ioc_data[SNXIocParse.SNX_PATTERN_KEY]
+        patternInfo = patternInfo.replace('[', '')
+        patternData = patternInfo.replace(']', '')
+        patternKey = patternData.split("=")[0].strip()
+        patternValue = patternData.split("=")[1].replace("'", '').strip()
+        ioc_type = None
+        if patternKey in FILE_HASHES_KEY_LIST:
+            ioc_type = patternKey
+        else:
+            ioc_type = patternKey.lower()
+        if ioc_type in SNX_TYPES_TO_DEMISTO_TYPES.keys():
```

Comment:
Could it be a case that you get a type that Demisto doesn't support?
And if so, how do you deal with it?

Reply:
Our threat feed data is in STIX 2.1 and if there is a type that demisto does not support we ignore the value.